libqtile/command/base.py

```diff
@@ -55,11 +57,114 @@ class CommandException(Exception):
 class CommandObject(metaclass=abc.ABCMeta):
     """Base class for objects that expose commands
 
-    Each command should be a method named `cmd_X`, where X is the command name.
+    Any command to be exposed should be decorated with
+    `@CommandObject.expose()` (classes that are not explicitly
+    inheriting from CommandObject will need to import the module)
     A CommandObject should also implement `._items()` and `._select()` methods
     (c.f. docstring for `.items()` and `.select()`).
     """
 
+    def __new__(cls, *args, **kwargs):
+
+        # Check which level of command object has been parsed
+        # This test ensures inherited classes don't stop additional
+        # methods from being exposed
```

Comment:
Please could you reword this for clarity? I'm getting a bit confused at "inherited classes". Are you referring to parent classes?

Reply:
I'll add some additional text.